### ISSUE TITLE
Switch service on dashboard should be underlined like a link

### ISSUE THREAD

**Issue by willeybryan** [cds-snc/notification-api#391]
**Current**
- Dashboards have the switch service link in blue on the top right side of the dashboard https://notification.alpha.canada.ca/services/e609f6b0-0390-45b0-aaae-f4cc92c713e4

**Desired**
For a11y reasons this should be underlined.
